File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/verification/pharmacy/shared/pharmacy-substore-requisition-verification.dto.ts
import { PharmacySubStoreRequisitionItemVerification_DTO } from "./pharmacy-substore-requisition-item-verification.dto";
import { PharmacyVerificationActor } from "./pharmacy-verification-actor.dto";

export class PharmacySubStoreRequisitionVerification_DTO {
    public RequisitionId: number;
    public RequisitionNo: number;
    public StoreId: number;
    public RequisitionDate: string;
    public RequisitionStatus: string;
    public CreatedBy: number;
    public CreatedOn: string;
    public RequisitionItems: PharmacySubStoreRequisitionItemVerification_DTO[];
    public CancelledBy?: number;
    public CancelledOn?: Date;
    public CancelRemarks: string;
    public CurrentVerificationLevel: number;
    public MaxVerificationLevel: number;
    public CurrentVerificationLevelCount: number;
    public VerificationStatus: string;
    public IsVerificationAllowed: boolean;
    public RequestedStoreName: string;
    public RequestedBy: string;
    public VerificationRemarks: string;
    public VerifierList: PharmacyVerificationActor[] = [];
    public TransactionType: string;

}

File: Code/Websites/DanpheEMR/Services/Verification/DTOs/Pharmacy/PharmacySubStoreRequisitionVerification_DTO.cs
using System.Collections.Generic;
using System;

namespace DanpheEMR.Services.Verification.DTOs.Pharmacy
{
    public class PharmacySubStoreRequisitionVerification_DTO
    {
        public int RequisitionId { get; set; }
        public int RequisitionNo { get; set; }
        public int StoreId { get; set; }
        public DateTime RequisitionDate { get; set; }
        public string RequisitionStatus { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public virtual List<PharmacySubStoreRequisitionItemVerification_DTO> RequisitionItems { get; set; }
        public int? CancelledBy { get; set; }
        public DateTime? CancelledOn { get; set; }
        public string CancelRemarks { get; set; }
        public int CurrentVerificationLevel { get; set; }
        public int MaxVerificationLevel { get; set; }
        public int CurrentVerificationLevelCount { get; set; }
        public string VerificationStatus { get; set; }
        public bool IsVerificationAllowed { get; set; }
        public string  VerificationRemarks { get; set; }
        public int? VerificationId { get; set; }
        public string TransactionType { get; set; }
    }
}
